### PR TITLE
Schedulling

### PR DESCRIPTION
**Pull request by coderwithattitude**
mobx issues fixed and all inputs rendered on confirm settings.

### CODE REVIEW

File: app/components/ScheduleWizard/AwaitingMining.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+class AwaitingMining extends Component {
+  state = {}
+  constructor(props){
+    super(props);
+  }
+  componentDidMount() {
+
+  }
```
Comment (Bagaric) [inline app/components/ScheduleWizard/AwaitingMining.js]: do we need `componentDidMount`? can be removed

File: app/components/ScheduleWizard/BountySettings.js [hunk 2]
```diff
@@ -1,45 +1,48 @@
...
 
-render() {
-  return (
-
-    <div id="bountySettings">
-
-      <div className="row">
-        <div className="col-md-4">
+  componentDidMount() {
+  }
```
Comment (Bagaric) [inline app/components/ScheduleWizard/BountySettings.js]: do we need `componentDidMount`? can be removed

File: app/components/Common/AwaitingMining.js [hunk 3]
```diff
@@ -61,6 +60,7 @@ class AwaitingMining extends Component {
   }
 
 render() {
+//  const props = this.props; .....uncomment when props is needed on this page
```
Comment (Bagaric) [inline app/components/Common/AwaitingMining.js]: Can be removed.

Comment (coderwithattitude) [reply app/components/Common/AwaitingMining.js]: @Bagaric removed, thanks for the feedback

File: app/components/ScheduleWizard/AwaitingMining.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
```
Comment (adibas03) [inline app/components/ScheduleWizard/AwaitingMining.js]: this file is to be deleted a more recent version exists in `./common`

File: app/components/ScheduleWizard/ConfirmSettings.js [hunk 5]
```diff
@@ -1,21 +1,27 @@
...
+@inject('scheduleStore')
+@inject('transactionStore')
+@observer
 class ConfirmSettings extends Component {
+  constructor(props){
+    super(props);
+  }
   state = {}
 
     componentDidMount() {
```
Comment (adibas03) [inline app/components/ScheduleWizard/ConfirmSettings.js]: uneeded componentDidMount

**Pull request merged**